dtmi/azure/iot/deviceupdate-1.json

```diff
@@ -0,0 +1,163 @@
+        {
+          "name": "compatPropertyNames",
+          "schema": "string"
+        },
+        {
+          "name": "deviceProperties",
+          "schema": {
+            "@type": "Map",
+            "mapKey": {
+              "name": "devicePropertyName",
+              "schema": "string"
+            },
+            "mapValue": {
+              "name": "devicePropertyValue",
+              "schema": "string"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "@id": "dtmi:azure:iot:deviceUpdate:serviceMetadata;1",
+      "@type": "Object",
+      "fields": [
+        {
+          "name": "workflow",
+          "schema": "dtmi:azure:iot:deviceUpdate:workflowMetadata;1"
+        },
+        {
+          "name": "updateManifest",
+          "schema": "string"
+        },
+        {
+          "name": "updateManifestSignature",
+          "schema": "string"
+        },
+        {
+          "name": "fileUrls",
+          "schema": {
+            "@type": "Map",
+            "mapKey": {
+              "name": "fileId",
+              "schema": "string"
+            },
+            "mapValue": {
+              "name": "downloadUri",
+              "schema": "string"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "@id": "dtmi:azure:iot:deviceUpdate:workflowMetadata;1",
+      "@type": "Object",
+      "fields": [
+        {
+          "name": "action",
+          "schema": "integer"
+        },
+        {
+          "name": "id",
+          "schema": "string"
+        },
+        {
+          "name": "retryTimestamp",
+          "schema": "dateTime"
+        }
+      ]
+    },
+    {
+      "@id": "dtmi:azure:iot:deviceUpdate:installResultMetadata;1",
+      "@type": "Object",
+      "fields": [
+        {
+          "name": "resultCode",
+          "schema": "integer"
+        },
+        {
+          "name": "extendedResultCode",
+          "schema": "integer"
+        },
+        {
+          "name": "resultDetails",
+          "schema": "string"
+        },
+        {
+          "name": "stepResults",
+          "schema": {
+            "@type": "Map",
+            "mapKey": {
+              "name": "stepIndex",
+              "schema": "string",
+              "comment": "The string format is step_{index}. For example, step_0, step_1, etc."
+            },
+            "mapValue": {
+              "name": "stepResult",
+              "schema": "dtmi:azure:iot:deviceUpdate:stepResultMetadata;1"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "@id": "dtmi:azure:iot:deviceUpdate:stepResultMetadata;1",
+      "@type": "Object",
+      "fields": [
+        {
+          "name": "resultCode",
+          "schema": "integer"
+        },
+        {
+          "name": "extendedResultCode",
+          "schema": "integer"
+        },
+        {
+          "name": "resultDetails",
+          "schema": "string"
+        }
+      ]
+    }
+  ],
+  "@context": "dtmi:dtdl:context;2"
```

Comment:
I'd suggest to move the `@context` to the first line in the interface

Reply:
Fixed it